sdk/python/tests/utils/auth_permissions_util.py

```diff
@@ -106,14 +107,52 @@ def default_store(
     return fs
 
 
+def start_feature_server(repo_path: str, server_port: int):
```

Comment:
shouldn't this go into the integration package, conftest module instead?

Reply:
current test is calling this method explicitly so can't move it to conftest.py as a fixture. I agree that there is some scope to abstract the code by adding it as a fixture but that is a low priority for now. Please let me know if you think it other way. I do have some other similar observations as well. I will create a separate jira ticket and track all my observations.